src/channels/ChallengeChannels/types.ts

```diff
@@ -1,12 +1,12 @@
 import { DidPublicKey } from '@kiltprotocol/types';
 
 export interface ChallengeInput {
-  dAppEncryptionKeyId: DidPublicKey['id'];
+  dAppEncryptionKeyId: DidPublicKey['uri'];
   challenge: string;
 }
 
 export interface ChallengeOutput {
-  encryptionKeyId: DidPublicKey['id'];
+  encryptionKeyId: DidPublicKey['uri'];
```

Comment:
Should we use this now?
```suggestion
  encryptionKeyId: DidResourceUri;
```